### PR TITLE
Adjust the way cm's priority is set

### PR DESCRIPTION
**Pull request by hjelmn**
:bot:assign: @jsquyres 
:bot:label:bug
:bot:milestone:v2.0.0

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-v1.8-pr/898/ for details.

File: ompi/mca/mtl/mxm/mtl_mxm_component.c [hunk 1]
```diff
@@ -130,7 +130,8 @@ static int ompi_mtl_mxm_component_register(void)
     free(runtime_version);
 #endif
 
-    param_priority = 100;
+    /* set high enought to defeat ob1's default */
```
Comment (jsquyres) [inline ompi/mca/mtl/mxm/mtl_mxm_component.c:133]: Comment is spelled wrong.  :-)

**Comment by jsquyres** [comment]
Minor nit about the comment spelled wrong, but otherwise :+1:

**Comment by hjelmn** [comment]
:bot:label:pushed-back

Added the pushed-back label until @jsquyres figures out the whole fake verbs provider issue in open-mpi/ompi#1060

**Comment by hppritcha** [comment]
@jsquyres  has the whole fake verbs provider issue been satisfactorily fixed in v2.x now?

**Pull request merged**